Sources/MapboxMaps/Foundation/Camera/FlyToAnimator.swift

```diff
@@ -0,0 +1,117 @@
+import UIKit
+
+internal class FlyToAnimator: NSObject, CameraAnimatorProtocol {
+
+    internal weak var delegate: CameraAnimatorDelegate?
+
+    internal var owner: AnimationOwner
+
+    internal var flyToInterpolator: FlyToInterpolator?
+
+    internal var animationDuration: TimeInterval?
+
+    internal private(set) var state: UIViewAnimatingState = .inactive
+
+    internal var startTime: Date?
+
+    internal var endTime: Date?
+
+    internal var finalCameraOptions: CameraOptions?
+
+    internal var animationCompletion: AnimationCompletion?
```

Comment:
could more of these be private?

Reply:
Good call, making them all `internal private(set)`